src/Lean/Parser/Command.lean

```diff
@@ -77,7 +77,7 @@ def «instance»       := leading_parser Term.attrKind >> "instance" >> optNamed
 def «axiom»          := leading_parser "axiom " >> declId >> ppIndent declSig
 /- As `declSig` starts with a space, "example" does not need a trailing space. -/
 def «example»        := leading_parser "example" >> ppIndent optDeclSig >> declVal
-def ctor             := leading_parser "\n| " >> ppIndent (declModifiers true >> rawIdent >> optDeclSig)
+def ctor             := leading_parser atomic (declModifiers true >> "\n| ") >> ppGroup (rawIdent >> optDeclSig)
```

Comment:
I'm not sure if using `atomic` here has any undesirable performance consequences.  Without the atomic, the inductive would eat the docstring of the following definition (and turn it into a constructor with a parse error).

Reply:
This is fine.